qna/DE-fiskaltrust-sandbox.md

```diff
@@ -11,14 +11,12 @@ How can I test my implementation?
 What is the fiskaltrust sandbox?
 
 ## Metadata tags
-lang-en, market-all, middleware, portal
+lang-en, market-de, middleware
```

Comment:
Should it be tagged with market-at and market-fr too?

Reply:
No! It is with this screenshot and information only valueable for the german market!